Add delimiter to titleCase in extend method (Issue 38)

Jao-Quin opened the pull request:
PR to fix Issue 38

jfstn commented:
Hello @Jao-Quin ! 👋

Can you add a test to the PR?

Jao-Quin commented:
Hey @jfstn ! Totally new to GitHub, so I'm not sure exactly the place/format for tests, but I'll learn a bit and get back to it!

jfstn commented:
Sure, take your time. You can find the current tests at `./test/validator.js`.

Check `package.json` scripts aswell.

Jao-Quin commented:
Added a test and ran it. First time using Jasmine - did I do it right?

jfstn commented on test/validator.js:
```diff
@@ -174,6 +174,14 @@ describe('Validator', () => {
             });
         });
+
+        const rules2 = { id: 'mongo_id:min=24,max=24' };
```
Instead of creating a new `rules2` object and a new `Validator` instance can you add that rule and a new property to the validation object above?

Thanks

jfstn commented:
Hey @Jao-Quin 👋 

Have you seen the code review?

Jao-Quin commented:
Hey @jfstn!

Sorry for slow response - I saw it and been meaning to do it; work's just been busy and I moved to a different area of the code.

There's another issue with the Numeric validator excluding scientific notation. Fixed it locally; will try to post a separate PR soon.

I'll come back to the validation soon as I'm planning a custom 'exists' checker to see if data is in the database. Not yet sure if it will be too site-specific or if it can be generalised for a PR.

jfstn commented:
Sure! Thank you for your response 😊

Jao-Quin commented:
Oof, sorry again for the long delay. Glad you were able to merge!

jfstn commented:
It’s fine. Thank you for your contribution. 🙂
